tasks/workflow/builder/builder.go

```diff
@@ -99,7 +99,12 @@ func setCancelPath(s *Step) {
 }
 
 func buildStepFromWFStep(kv *api.KV, deploymentID, wfName, stepName string, wfSteps map[string]*tosca.Step, visitedMap map[string]*visitStep) (*Step, error) {
-	wfStep := wfSteps[stepName]
+	var wfStep *tosca.Step
+	var ok bool
+	if wfStep, ok = wfSteps[stepName]; !ok {
```

Comment:
```suggestion
	if wfStep, ok := wfSteps[stepName]; !ok {
```

Reply:
Hum...but wfStep is a new locale var to the if bloc (declared and not used) except if I am wrong ?

Reply:
Yep you are right